src/utils/msb.nr

```diff
@@ -12,6 +12,15 @@ global MUL_DE_BRUIJN_BIT: [u32; 128] = [
     3, 26, 38, 44, n1, 56,
 ];
 
+global MUL_DE_BRUIJN_BIT_128: [u32; 128] = [
+    1, 14, 2, 15, 26, 20, 3, 16, 68, 80, 27, 21, 56, 50, 4, 17, 65, 96, 69, 81, 105, 99, 28, 22, 86,
+    90, 57, 51, 72, 42, 5, 126, 18, 66, 48, 94, 97, 84, 70, 124, 82, 122, 106, 100, 114, 108, 29,
+    23, 77, 102, 87, 91, 119, 116, 58, 52, 61, 110, 73, 37, 43, 31, 6, 127, 13, 25, 19, 67, 79, 55,
+    49, 64, 95, 104, 98, 85, 89, 71, 41, 125, 47, 93, 83, 123, 121, 113, 107, 76, 101, 118, 115, 60,
+    109, 36, 30, 12, 24, 78, 54, 63, 103, 88, 40, 46, 92, 120, 112, 75, 117, 59, 35, 11, 53, 62, 39,
+    45, 111, 74, 34, 10, 38, 44, 33, 9, 32, 8, 7, 128,
+];
+
 pub unconstrained fn get_msb64(x: u64) -> u32 {
```

Comment:
We don't need to leave the 64bit version around once we're confident that the 128 bit version is equivalent (we don't expose it outside of the crate).